controllers/feeds/getFeedChat.js

```diff
@@ -51,14 +51,14 @@ const isMediaOnlyMessage = (b) => {
   return isImagesIncluded && isMessageEmpty;
 };
 
-const getDetail = (req, b, id, anonymousId) => {
+const getDetail = (req, b, id, anonymousId, options = {}) => {
   const activity_id = b.reaction?.activity_id || b.id;
   const expired_at = b?.object?.expired_at || b.expired_at || null;
   const downvote = typeof b.object === 'object' ? b.object.reaction_counts?.downvotes : 0;
   const upvote = typeof b.object === 'object' ? b.object.reaction_counts?.upvotes : 0;
   const totalComment = typeof b.object === 'object' ? b.object.reaction_counts?.comment : 0;
   const childComment = typeof b.object === 'object' ? b.object?.latest_reactions?.comment : [0];
-  const message = getMessage(b);
+  const message = options?.getNewMediaMessage ? getMessage(b) : b.message || '';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getDetail` function has been modified to include an optional `options` parameter. This is a good change as it allows for more flexibility in the function's behavior. However, there seems to be a potential issue with the `message` variable assignment. If `options.getNewMediaMessage` is not set or false, and `b.message` is also undefined or empty, then `message` will be an empty string. This could potentially lead to issues downstream if the code expects `message` to always have a value. Consider adding a default message or handling this case where `message` might be an empty string.
```diff
- const message = options?.getNewMediaMessage ? getMessage(b) : b.message || '';
+ const message = options?.getNewMediaMessage ? getMessage(b) : (b.message || 'Default Message');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->